build.gradle

```diff
@@ -69,4 +69,17 @@ ext {
 
 test {
     useJUnitPlatform()
+}
+
+task restoreTopics(type: JavaExec) {
+    if (project.hasProperty("configPath")) {
+        println "Topic restore configuration path ["+project.getProperty("configPath")+"]"
+    }
```

Comment:
And if not?

Reply:
will skip this if because project.hasProperty will return error when configPath is not set
`Could not get unknown property 'configPath' for root project 'kafka-backup' `